packages/lib/constants.ts

```diff
@@ -9,6 +9,7 @@ export const WEBSITE_URL = process.env.NEXT_PUBLIC_WEBSITE_URL || "https://cal.c
 export const WEBSITE_DOMAIN = WEBSITE_URL.replace(/http(s):\/\//, "");
 export const APP_NAME = process.env.NEXT_PUBLIC_APP_NAME || "Cal";
 export const SUPPORT_MAIL_ADDRESS = process.env.NEXT_PUBLIC_SUPPORT_MAIL_ADDRESS || "help@cal.com";
+export const COMPANY_NAME = process.env.NEXT_PUBLIC_COMPANY_NAME || "help@cal.com";
```

Comment:
'Cal.com, Inc.' should be set as default

Reply:
thats true